xDrip/Components/Charts/Protocols/GlucoseChartGlucoseEntry.swift

```diff
@@ -19,3 +19,10 @@ protocol GlucoseChartGlucoseEntry {
     var date: Date { get }
     var severity: GlucoseChartSeverityLevel { get }
 }
+
+protocol GlucoseCurrentInfoEntry {
```

Comment:
This protocol doesn't relate to GlucoseChart, move it to a separate file. Also, move all `GlucoseCurrentInfoView` related files to `Home/Views/` folder, because it is used only there.